airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -562,8 +566,8 @@ class CheckoutSessionsLineItems(SingleEmptySliceMixin, IncrementalStripeStream):
 
     cursor_field = "checkout_session_expires_at"
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Just wondering, is adding these just for good measure? Don't see us passing any non-keyword args